cmd/cleanproto/Makefile

```diff
@@ -0,0 +1,8 @@
+build: cleanproto
```

Comment:
Hmmm... this will not rebuild if a file is there aleady?
I assume this is what you want? Otherwise you can merge the cleanproto line into build.

Adding `.PHONY` on top is also nice